iroh/src/node/builder.rs

```diff
@@ -146,6 +147,40 @@ impl From<Box<ConcurrentDiscovery>> for DiscoveryConfig {
     }
 }
 
+/// A server endpoint that does nothing. Accept will never resolve.
```

Comment:
why not use an `Option<BoxedServerEndpoint>` instead?

Reply:
It complicates the select! statement where you accept on it. This is basically just a boxed future::pending that does no harm whatsoever when disabled.